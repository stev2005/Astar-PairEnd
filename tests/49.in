CGATAGCCTGCCATAATGTTCACGTTTATGCCACAATCCGTGTAGTACGTATTCCGACTATACTCCTAGATGTGGCGGGACTTCACTCGGCCGGATTTACTTTGCCTTTGTCCTGCACGGATTGCTGAGGGCCCAGTTGGCCTGACAATGGTGCCTATAGCGAGTACGTTCCAATCGTCCGAAACC
15
GCTAGCACCAGTACACCCAGTAAAGAAGCCCTGCACGAGTCATCCTGGAGCTGCTACTGGCAGTCGATGGTTGGATCGGGCTCTACACAGTCGCCTTCTGAGCGAGGCAATCTCCTGCAGTTAGCCCAGAG
GACGGTAGTTAGACCGGCCCACTCGCGACATAAAGGTGCGGCAGGC
AGGCGTCACGGCTGATTAGCTAGCGTCCCGGCAATTTATAGCCCTGCGGTAGTGTGGCTTTCCACATAAGAGTCTGTACGTCATTTCCAACACGAGGTGGCGAATTGTATCTACGAATCCTGCAA
GGAAACGAC
ATCATGTTTGTTCCACTCCAGAGGATTTTACTAGTTATGTCGGGTTTGACGTCCCCACATCCTCTGATCTGGCCCAAAGACCTGGC
GGACTGACCTCCAATTCAACAAACGCAATCACTCGTTTAAGCCGGCCTCCCGGCTAGTAC
CGAGATCTTCCCTATACGCTTTCAGCGGCTTGCTAGTCCGTTTGTGGACTTATACCCTTTTGCAGGGCTATGTTAGCGGGGGTCGAGATGTGAATGGCTGCTAAGCGTTCGCTCGCCCCATATTCGGTTCCCAGCGTAGGCAAACGCGTTGGTCGATAATCCAGT
AGCGCGGCGTTACTTACGCTTCTTACAATACTGTGTCAAAAAACTTCACTAAATTAAAATACTGACGGCGGCGGTCCAGGTGTTCGTGGTCGCACCCTTTCCAATTCAATTTCGTTAGCTGGCTTGAACATTCATCTACTAATCGGATACGGATCTGGTTGTGTTCAGGGGGG
TCTAATCGGCCTATCATTTCTGTTACCGTATGCGGCGATACATGACGTACATT
TTAAGTACGCAAGGACGGGCAGGGCTTCGGCCCCGTCGCTACTGTAACGGGGACA
ATGGCTAGCGGGATCAAATTATAGCGACT
GTTAGAAGTCACTAAACAAATCTTTACACCATAATGATATAAATCATGAAGTCCGACAAGCCCCCAACAACACCAGGTAGTGGATACAACGCGTTTTTCTTGAATAGCTGATAGTTGATTGCAAATAAGCTGTTGGTCAT
AGAGCTAGTTAAAACTACGCAAAAGTTTTGTTATCGGCACCAGCATACAGGAGTACGAACGTATGCCCGCGTCAACTCGTTCAGAATGATTTTTG
ATGGACGCCGGCTCATGACGCAACTTAGGGAGCGACTGGCCCGAGGTATAGAAGCACGGTAGCGCCTAAGCCTTCCGCCCCTCGCGGGAACCTGTATGATAGATCGATTCAAA
GCAATACACAGAAGTACTGGTAATTTCATCCGTCGGGTTTTCATTTACGGTCGTCGGGGCTTTGCGCTCAGCCGACCAGATCCCAGTGACTACTGGCTCGTTTCGAGTAATTCCCCTATAGGATCGCGGGCCCAGTAATCAGACTCGTGCTAATTCCAATGCCTCTTGA
