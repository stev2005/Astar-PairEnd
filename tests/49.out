62
20
60
3
38
26
78
86
22
25
12
68
43
54
77
